e2etest/bgp_test.go

```diff
@@ -780,7 +846,7 @@ func validateFRRPeeredWithNodes(cs clientset.Interface, c *frrcontainer.FRR, ipF
 		framework.ExpectNoError(err)
 		err = frr.NeighborsMatchNodes(allNodes.Items, neighbors, ipFamily)
 		return err
-	}, 4*time.Minute, 1*time.Second).Should(BeNil())
+	}, 4*time.Minute, 1*time.Second).Should(chkFunc())
```

Comment:
Checking that we are returning errors here may hide bugs, in general I'd rather be explicit in what we are looking for in a test and not checking for random failures.

As a side note, if we want to check a negative condition, we should use `Consistently` because here you may get a transitient error and exit immediately. When checking if an error persist, we must check it for x time.

Reply:
I used same peers with and w/o no peer community to proof the update message carry the new attribute

Reply:
you should gomega.Consistently instead of Eventually to check errors happening. Checking for random errors may cause false positives.

Reply:
not needed anymore